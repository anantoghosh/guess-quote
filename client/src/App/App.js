import React from "react";
import PropTypes from "prop-types";
import "./App.css";

const App = ({ children }) => {
  return <div className="App">{children}</div>;
};

App.propTypes = {
  children: PropTypes.node.isRequired
};

App.defaultProps = {
  children: <div>Loading</div>
};

export default App;
