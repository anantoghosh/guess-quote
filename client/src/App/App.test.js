import React from "react";
import { shallow } from "enzyme";
import App from "./App";

describe("<App />", () => {
  test("Match Snapshot", () => {
    const el = shallow(<App />);
    expect(el).toMatchSnapshot();
  });
});
