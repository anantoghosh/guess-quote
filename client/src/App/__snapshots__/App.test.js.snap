// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> Match Snapshot 1`] = `
<div
  className="App"
>
  <div>
    Loading
  </div>
</div>
`;
